decksite/controllers/api.py

```diff
@@ -143,6 +143,40 @@ def decks_api() -> Response:
     resp.set_cookie('page_size', str(page_size))
     return resp
 
+@APP.route('/api/scraper/')
```

Comment:
I'd much rather you use the modern API stuff provided by [restx](https://flask-restx.readthedocs.io/en/latest/) (See /api/decks/ as an example).
I've been slowly going through and porting all of the API endpoints, and adding new old-style endpoints is not something I want to be doing

Reply:
I think I made this work now